src/Xunit.NetCore.Extensions/Xunit.NetCore.Extensions.csproj

```diff
@@ -7,6 +7,7 @@
     <IsTestProject>false</IsTestProject><!-- while this project references xunit it isn't itself a test -->
   </PropertyGroup>
   <ItemGroup>
+    <PackageVersion Include="xunit.core" Version="$(XUnitVersion)" />
     <PackageReference Include="xunit.core" />
```

Comment:
@GangWang01 this is a clever workaround. But please comment it since you're not _supposed_ to put PackageVersion items in individual projects :)

Reply:
Turns out it's possible to simplify this to a single element by using `VersionOverride` instead, so I did that + modified the comment with @ViktorHofer's feedback.